console/ui/styles.css

```
@@ -170,3 +170,1689 @@
+  font-size: 11px;
+}
+[data-iii-ui="console"] .console-catalog-result-head > :first-child {
+  display: inline-flex;
+  align-items: center;
+  gap: 6px;
+}
+[data-iii-ui="console"] .console-catalog-result-head .dot {
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)